.github/workflows/main.yaml

```diff
@@ -0,0 +1,68 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  test-blockdiag:
+    runs-on: ubuntu-18.04
+
+    steps:
+    # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
```

Comment:
Not sure if this comment is really helpful, I think the task name is explicit enough no?